docker-compose.yaml

```
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  stable-diffusor:
+    # TODO: exchange this image with an image of CompVis
+    image: guestros/stable-diffusion:latest
+    build: .
```

Given that you're setting up miniconda to run on a specific architecture, might as well put this in there for those of us that have M1s, etc.

```suggestion
    build: .
    platform: linux/amd64
```

---

hi hkiang01, good suggestion, i am trying to add arm as well, even though I don't think it will compile. i will test it locally, and otherwise add your suggestion

---

> I don't think it will compile

It works on my machine, and it should work on any machine that can ultimately build linux/amd64 Docker images, which include at least most modern 64-bit machines and M1/M2